bootloader/src/pyi_utils.c

```diff
@@ -850,21 +855,44 @@ set_systemd_env()
 pid_t child_pid = 0;
 
 static void
-_signal_handler(int signal)
+_ignore_signal(int signum)
```

Comment:
Rename to something telling this is a handler, too. E.g. `ignoring_signal_handler`. Also a comment would be nice.

Reply:
What would the comment say that isn't already obvious from the function name and log message?